### ISSUE TITLE
Build bundle error for RN 0.49

### ISSUE THREAD

**Issue by GROL** [realm/realm-js#1380]
Red screen with text:

```
The development server returned response error code: 500
URL: http://10.0.2.2:8081/index.bundle?platform=android&dev=true&minify=false

Body:
{"message":"TransformError: C:\\Projects\\_rn\\rn49\\node_modules\\realm\\lib\\index.js: require() must have a single string literal argument","type":"TransformError","lineNumber":0,"description":"","errors":[{"description":"","lineNumber":0}]}
processBundleResult
    BundleDownloader.java:225
access$100
    BundleDownloader.java:40
onResponse
    BundleDownloader.java:192
execute
    RealCall.java:135
run
    NamedRunnable.java:32
runWorker
    ThreadPoolExecutor.java:1112
run
    ThreadPoolExecutor.java:587
run
    Thread.java:841
```

## Version of Realm and Tooling

- Realm JS SDK Version: 1.12.0
- Node or React Native: 0.49

**Comment by kneth** [comment]
Is it the same as https://github.com/realm/realm-js/issues/1294?

**Comment by blagoev** [comment]
no it seems like fail in bundle step. it's different

**Comment by adyanced** [comment]
Same issue after upgrading to RN0.49.1

**Comment by ashwinphatak** [comment]
Seems like https://github.com/realm/realm-js/issues/1342?

**Comment by giladno** [comment]
Latest version of realm still fails with the same error. Any estimate on a fix?

**Comment by ashwinphatak** [comment]
It's been fixed in https://github.com/realm/realm-js/pull/1346 and will be released soon. Closing as duplicate of #1342.

**Comment by niocncn** [comment]
when new version would be released?)

**Comment by blagoev** [comment]
@niocncn it is already released in v1.13.0
